README.md

```diff
@@ -31,3 +44,51 @@ imap_data_access.download("imap/mag/l0/2024/01/imap_mag_l0_raw_202040101_2024010
 # Upload a calibration file that exists locally
 imap_data_access.upload("imap/mag/calibration/test_calibration.txt")
 ```
+
+## Configuration
+
+### Data directory
+
+The folder structure for data files within the IMAP SDC is rigidly
+defined, so the data access will mimic that structure to make sure
+all data is stored in the same heirarchical structure as the SDC.
+This will enable seamless transition between a user's local system
+and the SDC.
+
+A user's root data location can be specified as an environment
+variable ``IMAP_DATA_DIR`` or through a configuration dictionary
+within the package itself ``imap_data_access.config["DATA_DIR"]``.
+
+### Data Access URL
+
+To change the default URL that the package accesses, you can set
+the environment variable ``IMAP_DATA_ACCESS_URL`` or within the
+package ``imap_data_access.config["DATA_ACCESS_URL"]``. The default
+is the development server ``https://api.dev.imap-mission.com``.
+
+## Troubleshooting
+
+### Network issues
+
+If you encounter SSL errors similar to the following:
+
+```text
+urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>
+```
+
+That generally means the Python you're using is not finding your system's root
+certificates properly. This means you need to tell Python how to find those certificates
+with the following potential solutions.
+
+1. Upgrade the certifi package
+
+    ```bash
+    pip install --upgrade certifi
+    ```
+
+2. Install system certificates
+    Depending on the Python version you installed the program with the command will look something like this:
+
+    ```bash
+    /Applications/Python\ 3.10/Install\ Certificates.command
+    ```
```

Comment:
Two other troubleshooting cases we could add:

1. `urllib.error.HTTPError: HTTP Error 502: Bad Gateway` - I came across this when I tried a download, I think because the AWS `dev` stacks are currently being re-deployed? But it might be good to add a note that if users see this, to contact the SDC.
2. `FileNotFoundError` could indicate that the user does not have the data directory set up properly.

Reply:
Added both of these, all still under Networking. Not sure if we want to move them around at all or put differnet sub-headings.